bitnami/wavefront/README.md

```diff
@@ -0,0 +1,249 @@
+
+#### Collector parameters
+
+| Parameter                                  | Description                                                                      | Default                              |
+|--------------------------------------------|----------------------------------------------------------------------------------|--------------------------------------|
+| `collector.enabled`                        | Setup and enable the Wavefront collector to gather metrics                       | `true`                               |
+| `collector.image.registry`                 | Wavefront collector Image registry                                               | `docker.io`                          |
+| `collector.image.repository`               | Wavefront collector Image name                                                   | `bitnami/wavefront-collector`        |
+| `collector.image.tag`                      | Wavefront collector Image tag                                                    | `{TAG_NAME}`                         |
+| `collector.image.pullPolicy`               | Image pull policy                                                                | `IfNotPresent`                       |
+| `collector.image.pullSecrets`              | Specify docker-registry secret names as an array                                 | `nil`                                |
+| `collector.useDaemonset`                   | Use Wavefront collector in Daemonset mode                                        | `true`                               |
+| `collector.maxProx`                        | Max number of CPU cores that can be used (< 1 for default)                       | `0`                                  |
+| `collector.logLevel`                       | Min logging level (info, debug, trace)                                           | `info`                               |
+| `collector.interval`                       | Default metrics collection interval                                              | `60s`                                |
+| `collector.flushInterval`                  | How often to force a metrics flush                                               | `10s`                                |
+| `collector.sinkDelay`                      | Timout for exporting data                                                        | `10s`                                |
+| `collector.useReadOnlyPort`                | Use un-authenticated port for kubelet                                            | `false`                              |
+| `collector.useProxy`                       | Use a Wavefront Proxy to send metrics through                                    | `true`                               |
+| `collector.proxyAddress`                   | Non-default Wavefront Proxy address to use, should only be set when `proxy.enabled` is false | `nil`                    |
```

Comment:
Isn't it auto-calculated when `proxy.enabled` is `true`?

Reply:
It is needed only when `proxy.enabled` is false, by default the proxy is enabled and it is auto-calculated but if you disable the bundled proxy to use an existing one you need to specify here the address